cmd/build.go

```diff
@@ -38,7 +39,13 @@ func Build(ctx context.Context) *cobra.Command {
 	var secrets []string
 
 	cmd := &cobra.Command{
-		Use:   "build [PATH]",
+		Use: "build [PATH]",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) >= 2 {
+				return errors.New("okteto build requires one or no argument")
```

Comment:
Can we have the error in `errors.go` to make it easier to review CLI error messages by other teams?

Reply:
could we use `cobra.MaximumNArgs(1)`?